app/services/site_channel_verifier.rb

```diff
@@ -87,33 +56,41 @@ def verified_channel_post_verify
       PublisherMailer.verification_done_internal(verified_channel).deliver_later
     end
     SlackMessenger.new(message: "*#{verified_channel.publication_title}* verified by #{verified_channel.publisher.name} (#{verified_channel.publisher.email}); id=#{verified_channel.id}").perform
+
+    # Let eyeshade know about the new Publisher
+    begin
+      PublisherChannelSetter.new(publisher: @publisher).perform
+    rescue => e
+      # TODO: Retry if it fails
+      require "sentry-raven"
+      Raven.capture_exception(e)
+    end
   end
 
-  def verify_offline_publisher_id
-    Rails.logger.info("PublisherVerifier offline by #{channel.details.verification_method}")
+  def verify_publisher_id
+    Rails.logger.info("PublisherVerifier by #{channel.details.verification_method}")
 
     case channel.details.verification_method
       when "dns_record"
```

Comment:
I'd prefer if these were constants so pushing into staging would throw an error if there was a typo.

Reply:
This was existing code so I opted not to change it. However, I agree with the sentiment to constantize and will update the code.

Reply:
just kidding i'm not sure the optimal way to do this